### ISSUE TITLE
Fail to load plugin: GEmojiSharp

### ISSUE THREAD

**Issue by shap-po** [hlaueriksson/GEmojiSharp#25]
When starting PowerToys, I get the following message:

```
Power Toys Run - Plugin Loading Error

Fail to load plugin: GEmojiSharp
Please report the bug to https://aka.ms/powerToysReportBug. (For
third-party plugins, please contact the plugin author.)
```

![image](https://github.com/hlaueriksson/GEmojiSharp/assets/50052124/e6376fb1-110f-44a6-8917-be057677d76f)

**Comment by shap-po** [comment]
The issue appeared after the last PT update (v0.75.0)

**Comment by romanlex** [comment]
Can confirm issue
I get this error with 0.75.1

**Comment by hlaueriksson** [comment]
This bug is due to Group Policy Object (GPO) rules that was introduced in PowerToys `v0.75.0`:

- https://learn.microsoft.com/en-us/windows/powertoys/grouppolicy

> All plugins have to provide its plugin ID as static property in its Main method

- [Release v0.75.0](https://github.com/microsoft/PowerToys/releases/tag/v0.75.0#:~:text=All%20plugins%20have%20to%20provide%20its%20plugin%20ID%20as%20static%20property%20in%20its%20Main%20method)

This error can be found in the logs:

```
Can't validate plugin ID of plugin <GEmojiSharp> in C:\Users\<user>\AppData\Local\Microsoft\PowerToys\PowerToys Run\Plugins\GEmojiSharp\GEmojiSharp.PowerToysRun.dll: The static property <Main.PluginID> was not found.
```

I will release a fix shortly.

**Comment by hlaueriksson** [comment]
@shap-po thanks for reporting this issue!

You can try out a fix in version `3.1.2` of `GEmojiSharp.PowerToysRun`

Download and install; https://github.com/hlaueriksson/GEmojiSharp/releases/tag/v3.1.2

**Comment by romanlex** [comment]
All work perfectly
